playbooks/provisioning/openstack/provision-openstack.yml

```diff
@@ -14,7 +14,7 @@
         - cinder_hosted_registry_size_gb is defined
     - role: static_inventory
       when: openstack_inventory|default('static') == 'static'
-      inventory_path: "{{ openstack_inventory_path|default(inventory_dir) }}"
+      inventory_path: "{{ openstack_inventory_path|default(lookup('env','PWD') + '/inventory') }}"
```

Comment:
/me shakes fist at Ansible.

The default will now only work when the directory is in `$PWD/inventory`. Previously, it worked for any location. But I can't figure out how to do that under the new rules so let's do this.